website/docs/r/express_route_circuit_peering.html.markdown

```diff
@@ -82,6 +92,14 @@ A `microsoft_peering_config` block contains:
 * `customer_asn` - (Optional) The CustomerASN of the peering
 * `routing_registry_name` - (Optional) The RoutingRegistryName of the configuration
 
+A `ipv6_peering` block contains:
+
+* `microsoft_peering_config` - (Required)  A `microsoft_peering_config` block as defined above.
```

Comment:
I don't see this block defined anywhere?

Reply:
Rename `microsoft_peering_config` to `microsoft_peering` and  add a new `microsoft_peering` block description